test/cpp/interop/client.cc

```diff
@@ -120,6 +120,9 @@ ABSL_FLAG(int32_t, soak_min_time_ms_between_rpcs, 0,
 ABSL_FLAG(int32_t, iteration_interval, 10,
           "The interval in seconds between rpcs. This is used by "
           "long_connection test");
+ABSL_FLAG(int32_t, soak_request_size, 271828, "The request size in a soak RPC");
```

Comment:
nit: let's mention in the comments that by default, sizes are configured as per the "large unary" spec

Reply:
Done.